### PR TITLE
Improve dropdown handling for the mobile main navigation.

### PR DESCRIPTION
**Pull request by fklein-lu**
- Avoid passing HTML from PHP to Javascript.
- Create DOM elements instead of relying on string concatenation.
- Escape screen reader text with `.text()`.

### CODE REVIEW

**Comment by karmatosed** [comment]
Can you update this please, as its now out of sync.

File: js/functions.js [hunk 1]
```diff
@@ -20,12 +28,14 @@
 		container.find( '.menu-item-has-children' ).attr( 'aria-haspopup', 'true' );
 
 		container.find( '.dropdown-toggle' ).click( function( e ) {
-			var _this = $( this );
+			var _this            = $( this ),
```
Comment (valendesigns) [inline js/functions.js]: The space after the param needs to be removed.

File: js/functions.js [hunk 2]
```diff
@@ -20,12 +28,14 @@
...
+				screenReaderSpan = _this.find( '.screen-reader-text');
```
Comment (valendesigns) [inline js/functions.js]: There should be a space after the last quotation mark.

**Comment by valendesigns** [comment]
@fklein-lu Could you fix the above issues and whatever conflicts there are between the PR and master?

**Comment by fklein-lu** [comment]
I'll create a new PR, because fixing the conflicts will take more time than creating a new PR.

**Pull request closed without merging**